service.go

```diff
@@ -90,17 +101,56 @@ func (s *Service) Catch(msg *Message, err error, body []byte) error {
 		return err
 	}
 
+	return s.newChainProcess(msg, b, chain)
+}
+
+// Count starts set count chain. Initial data for chain is object with key `all` containing total count of elements
+func (s *Service) Count(msg *Message, count int) error {
+	_, nextElement, err := s.chainCursor(msg)
+	if err != nil {
+		return err
+	}
+	if len(nextElement.SetCount) == 0 {
+		return ErrorNextNoCount
+	}
+
+	data := struct {
```

Comment:
please, specify structure for the count message. It will be required for Unmarshal.